src/services/languageFacts.ts

```diff
@@ -783,7 +783,7 @@ let atDirectiveList: IEntry[];
 export function getAtDirectives(): IEntry[] {
 	if (!atDirectiveList) {
 		atDirectiveList = [];
-		for (let i = 0, len = atDirectives.length; i < len; i++) {
+		for (let i = 0; i < atDirectives.length; i++) {
```

Comment:
Why changing this?

Reply:
Originally I put `scss.ts` together with `data/browsers.ts`, and wrote a function `getScssAtDirectives` right below and fixed this declaration of one more temporary variable. Later I decided `scssCompletion.ts` is the better place to put it.